packages/components/src/toggle-control/style.scss

```diff
@@ -10,4 +10,9 @@
 		display: block;
 		margin-bottom: 4px;
 	}
+
+	// This overrides a border style that is inherited from parent checkbox styles.
+	.components-form-toggle__input {
```

Comment:
Since this styles `components-form-toggle__input` -- should the styling actually go to `packages/components/src/form-toggle/style.scss` so it'd also apply to `FormToggle` when not wrapped in a `ToggleControl`? Cf https://github.com/Automattic/wp-calypso/issues/28701

Reply:
Let me take a look as I address the other item from @simison, thanks!